butterfree/migrations/migration.py

```diff
@@ -0,0 +1,36 @@
+"""Migration entity."""
+
+from abc import ABC, abstractmethod
+from typing import Any, Callable
+
+
+class Migration(ABC):
+    """Abstract base class for Migrations."""
+
+    @abstractmethod
+    def create_query(self) -> Any:
+        """Create a query regarding a data source.
+
+        Returns:
+            Schema object.
+
+        """
+
+    def validate_schema(self, schema_object: Callable) -> Any:
```

Comment:
schema_object is an `object` (as the name says), not a `Callable`

Reply:
I've proposed a type for ```schema_object```, what do you think?